src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -27,7 +27,7 @@ public static void CompileDeepTree_NoStackOverflow(bool useInterpreter)
             Assert.Equal(n, f());
         }
 
-        [Theory, ActiveIssue("https://github.com/dotnet/corefx/issues/15533"), ClassData(typeof(CompilationTypes))]
+        [Theory, ClassData(typeof(CompilationTypes))]
```

Comment:
Passing 1 will cause it to use the system-defined minimum stack size (typically 16 KB on Unix). Requesting 64 KB should be portable.